### PR TITLE
Feature/storybook setup

### PR DESCRIPTION
**Pull request by Charity10**

### CODE REVIEW

**Comment by GDesigns49** [comment]
I had the opportunity to review this project, and here are my feedback points:

The README.md file is exceptional, providing a clear and comprehensive overview of the project's purpose and functionality.

I commend you for the excellent use of React's functional components and hooks. Your code structure is clean and organized.

Additionally, I appreciate your choice of variable and function names.

Your CSS code demonstrates a deep understanding of responsive design principles.

Overall, I am impressed with the quality of your work and attention to detail. Keep it up @Charity10 @JanetEne  @damyl4sure

File: src/stories/Page.jsx
Comment (meekg33k) [inline src/stories/Page.jsx:1]: Please delete this file, we don't too many examples.

File: src/stories/Introduction.mdx [hunk 1]
```diff
@@ -0,0 +1,213 @@
+import { Meta } from '@storybook/blocks';
```
Comment (meekg33k) [inline src/stories/Introduction.mdx:1]: This is file for now, but in the future we will edit this file.

File: src/stories/Header.stories.js
Comment (meekg33k) [inline src/stories/Header.stories.js:1]: Please delete this file, we don't need too many examples.

File: src/stories/Header.jsx
Comment (meekg33k) [inline src/stories/Header.jsx:1]: Please delete this file, we don't need too many examples.

File: src/stories/Button.stories.js
Comment (meekg33k) [inline src/stories/Button.stories.js:1]: This file can stay as an example.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "name": "storybook-setup",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "start": "storybook dev -p 6006",
```
Comment (meekg33k) [inline package.json:11]: Let's update this script from `start` to `start-storybook`

**Comment by ArafatAbdussalam** [comment]
Hi  @JanetEne @GDesigns49 @Adejuwon-00  @Charity10 . I will like to inform everyone involved in this project that I made some changes as instructed by @meekg33k in the PR review. 

Here is a link to the commits made to the project:

1. https://github.com/growwithuduak/geek-devs-frontend/commit/3597c7ce933b9a6f7403599d88012f96041bd560

2. https://github.com/growwithuduak/geek-devs-frontend/commit/6ba762438e46439d4c4def0390b013f4a0aaddc4

**Review by gwa-admin — APPROVED** [review]

**Review by ArafatAbdussalam — APPROVED** [review]

**Review by GDesigns49 — APPROVED** [review]

**Pull request merged**